de_workloads/data_processing/silver/spark_jobs/silver_dq.py

```diff
@@ -1,17 +1,13 @@
-from pysparkle.pysparkle_cli import cli
+import logging
 
+from pysparkle.data_quality.main import data_quality_main
+from pysparkle.logger import setup_logger
 
-def call_pysparkle_entrypoint():
-    """Execute data quality activity in Pysparkle."""
-    cli(
-        [
-            "data-quality",
-            "--config-path",
-            "data_processing/silver/data_quality/silver_dq.json",
-        ],
-        standalone_mode=False,
-    )
+CONFIG_CONTAINER = "config"
+
+logger_library = "pysparkle"
 
 
 if __name__ == "__main__":
-    call_pysparkle_entrypoint()
+    setup_logger(name=logger_library, log_level=logging.INFO)
+    data_quality_main(config_path="data_processing/silver/data_quality/silver_dq.json", container_name=CONFIG_CONTAINER)
```

Comment:
define as constant next to `CONFIG_CONTAINER`?

Reply:
done